lib/plugins/aws/lib/naming.js

```diff
@@ -163,8 +163,11 @@ module.exports = {
       ''
     )}`;
   },
-  getLambdaOnlyVersionLogicalId(functionName) {
-    return `${this.getNormalizedFunctionName(functionName)}LambdaVersion`;
+  getLambdaProvisionedConcurrencyVersionLogicalId(functionName) {
```

Comment:
We should test this with services which have functions with long function names as the suffix already counts 21 characters.

Reply:
This PR is in deep draft state (not yet issued for review)

Reply:
I know. Just wanted to raise awareness because we had a couple of issues with long names in the past.

Reply:
Sure, thank you @pmuens ! 👍

Reply:
@pmuens I've just double checked, and it's guaranteed to be shorter than lamba version logical id which we already use. Therefore I assume it's safe

Reply:
Great. Thanks for looking into that @medikoo :+1: